src/styles/utils/_vars.scss

```diff
@@ -1 +1,4 @@
 $c-gray: #eee;
+
+$tablet: 744px;
```

Comment:
whole tablet adaptive styles are way off the desired layout:
![Snapshot 2022-04-15 14 12 49](https://user-images.githubusercontent.com/39308717/163564733-a7476d79-8cf0-4e34-b78b-4c9c8bf7a50e.png)
more along the code